OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -901,6 +914,8 @@ public void onSaveInstanceState(Bundle outState) {
             bundle.putParcelable(Router.EXTRA_COURSE_UPGRADE_DATA, courseUpgradeData);
         if (courseComponentId != null)
             bundle.putString(Router.EXTRA_COURSE_COMPONENT_ID, courseComponentId);
+        if (screenName != null)
```

Comment:
IMO, I guess we don't need to save the `screenName`

Reply:
removed.